src/d3a/d3a_core/simulation.py

```diff
@@ -281,6 +281,9 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
                     if console is not None:
                         self._handle_input(console, diff_slowdown)
 
+                if ConstSettings.GeneralSettings.RUN_REAL_TIME:
+                    sleep(tick_lengths_s - tick_length)
```

Comment:
took me a wehil to understand this line because of the naming of `tick_length` AFAIU this is the realtime time difference between the start and the end of the tick. How about renaming it to `realtime_tick_length`  or `real_tick_length` ?

Reply:
Thanks, renaming it